__tests__/endpoint.test.js

```diff
@@ -237,3 +239,75 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+describe("PATCH: /api/articles/:article_id", () => {
+  test("the endpoint will successfully increment the article on a given ID by updating the votes on that article ", () => {
+    const newVote = {
+      incVotes: 10,
+    };
+
+    return request(app)
+      .patch("/api/articles/1")
+      .send(newVote)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body).toMatchObject({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 110,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      });
+  });
+  test("the endpoint will successfully decrement the article on a given ID by updating the votes on that article ", () => {
+    const newVote = {
+      incVotes: -100,
+    };
+
+    return request(app)
+      .patch("/api/articles/1")
+      .send(newVote)
+      .expect(201)
```

Comment:
`200`, `201` would be for when we create new content rather than update old content